.appveyor.yml

```diff
@@ -0,0 +1,133 @@
+version: '{branch}-{build}'
+os: Visual Studio 2015
+clone_folder: C:\projects\oiio
+test: off
+configuration:
+    - Release
+    - Debug
+platform:
+    - x64
+# branches:
+#   only:
+#     - master
+environment:
+  matrix:
+    # - CMAKE_PLATFORM: "Visual Studio 14 2015"
+    #   PYTHON_DIR: "C:\\Python27"
+    - CMAKE_PLATFORM: "Visual Studio 14 2015 Win64"
+      PYTHON_DIR: "C:\\Python27-x64"
+      tbs_arch: "x64"
+      tbs_tools: "msvc14"
+      tbs_static_runtime: 0
+
+install:
+  - cinstall: python
+  - cinst ninja
```

Comment:
Haha :-)

AFAICT from a search, `cinst` is a batch command alias of `choco install`, to install ninja via the chocolatey package manager.
When lines inside the `install` block are prefixed with `<interpreter>:` that's the script interpreter used to run them.  Presumably `cinstall` is the appveyor alias of chocolatey install... not sure though.